parodos-model-api/src/main/java/com/redhat/parodos/workflow/exception/MissingParameterException.java

```diff
@@ -13,16 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.redhat.parodos.workflows.common.exception;
+package com.redhat.parodos.workflow.exception;
 
 /**
- * Thrown when the WorkContext is missing a WorkParameter required by a Task
- * 
+ * Thrown when the @see WorkContext is missing a @see WorkFlowTaskParameter
+ *
  * @author Luke Shannon (Github: lshannon)
  *
  */
 public class MissingParameterException extends Exception {
+
+	private static final long serialVersionUID = 1L;
```

Comment:
can be removed

Reply:
Its unlikely that someone would serialize an exception. But, I personally like to set the serialVersionUID to 1L on objects so in the event the object model changes, and someone has a an older version of the object serialized (ie: an in-memory cache would do this) reading the stored object value into the new version of the object will not result in an serialization error.  On the flip side, we ensure our objects are backwards compatible.